concepticondata/conceptlists.tsv

```diff
@@ -349,3 +349,4 @@ Chen-2019-171	Chen, I-Hsuan and Zhao, Qingqing and Long, Yunfei and Lu, Qin and
 Speed-2017-485	Speed, Laura J. and Majid, Asifa	2017		485	ratings	English	Dutch	https://doi.org/10.3758/s13428-017-0852-3	Speed2017		The list presents modality exclusivity norms for 485 Dutch nouns rated on visual, auditory, haptic, gustatory, and olfactory associations. The authors also collected ratings of emotion (valence, dominance, and arousal) to assess its role in perceptual space simulation.		
 Gallatin-1836-180	Gallatin, Albert	1836		180	basic, areal, historical	English	Native languages in North America		Gallatin1836	Gallatin1836	This list was used in Gallatin’s groundbreaking ethnographic work on the native languages in North America, particularly in the areas west and north-west of the Rocky Mountains. The selection process for items on the list was centered around the idea of showing the differences between the different native languages. As such, the glosses were built from basic vocabulary as well as very general concepts, thought to exist in the majority of languages under investigation.	307-367	
 Hale-1846-179	Hale, Horatio	1846		179	basic, areal, historical	English	Native languages in the north-western United States		Hale1846	Hale1846	This list was used to collect basic vocabularies of the native tribes living west of the Rocky Mountains (up to the Pacific coast) in what was formerly known as Oregon Country. The items in the list are derived from [Albert Gallatin’s work](:bib:Gallatin1836) on basic vocabularies in this area.	570-629	
+Zalizniak-2020-2591	Zalizniak, Anna A. and Smirnitskaya, Anna and Russo, Maksim and Mikhailova, Tatiana and Bobrik, Marina and Gruntov, Ilya and Orlova, Maria and Bibaeva, Maria and Voronov, Mikhail	2020		2591	norms, historical	English	Global	http://datsemshift.ru/	Zalizniak2020		The list presents the most recent version of the database of semantic shifts (Version from 07/10/2020). The aim of the Database of Semantic shifts is to collect and classify semantic shifts attested in the languages of the world. While mapping the list, we ignored glosses with <>.		
```

Comment:
> The list presents the most recent version of the database of semantic shifts (Version from 07/10/2020).

should be rather:

> The list presents the most recent version of the Database of Semantic Shifts, from which the major shifts were automatically extracted on 07/10/2020. 

Also note: 

> The aim of the Database of Semantic shifts 

should be

> The aim of the Database of Semantic Shifts 

(should always be capitalized). 

And please rewrite:

> While mapping the list, we ignored glosses with <>.

As 

> The original data contains some ambiguous glosses enclosed in &gt; and &lt; symbols, which we deliberately ignored here.

Maybe also add a short statement on the mapping principle (ambiguous cases, or similar).